core/templates/dev/head/pages/exploration_player/PlayerServices.js

```diff
@@ -369,9 +389,16 @@ oppia.factory('oppiaPlayerService', [
           exploration.isInteractionInline(oldStateName));
 
         if (!_editorPreviewMode) {
+          var isFirstHit = false;
```

Comment:
var isFirstHit = Boolean(visitedStates.indexOf(newStateName) === -1);

(The Boolean cast is superfluous, but it's fine to have it for clarity if you like.)

Reply:
Done.